scripts/dotnet-cli-build/Utils/Dirs.cs

```diff
@@ -22,7 +22,9 @@ public static class Dirs
         public static readonly string Stage2 = Path.Combine(Output, "stage2");
         public static readonly string Stage2Compilation = Path.Combine(Output, "stage2compilation");
         public static readonly string Stage2Symbols = Path.Combine(Output, "stage2symbols");
-        public static readonly string Corehost = Path.Combine(Output, "corehost");
+        public static readonly string CorehostLatest = Path.Combine(Output, "corehost", "latest");
```

Comment:
May be just use Corehost but call it latest to keep signing working or it can be updated to read from this folder instead?